drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/current_living_situation.rb

```diff
@@ -16,13 +16,13 @@ class CurrentLivingSituation < GrdaWarehouse::Hud::Base
 
     def self.hmis_validations
       {
-        CurrentLivingSituation: [
+        EnrollmentID: [
           class: HmisCsvValidation::NonBlank,
         ],
         InformationDate: [
           class: HmisCsvValidation::NonBlank,
         ],
-        UserID: [
+        CurrentLivingSituation: [
           class: HmisCsvValidation::NonBlank,
         ],
         DateUpdated: [
```

Comment:
We aren't checking these fields consistently -- is there a rationale as to when they get checked? Maybe push them up to `GrdaWarehouse::Hud::Base` and make all the hmis_validation methods be `super.merge(...)`?

Reply:
This one bit us earlier, so it's here, but generally, we don't care if there's a valid UserID